runtime/tests/Test_OpenQasmDevice.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include "MemRefUtils.hpp"
 
-#include "ExecutionContext.hpp"
+#include "HostExecutionContext.hpp"
```

Comment:
what are your reasons behind changing the name of the runtime's execution context manager to 'host'?

Reply:
This change can be reverted. I want to keep a separation between the host and the device execution context. While working on this, I thought that for example, all qubit registers can be deallocated by the device execution context upon device release. I admit that after I looked at it, this commit can be reverted until we have that separation.

Reply:
Reverted the change here: https://github.com/PennyLaneAI/catalyst/pull/470/commits/5f963666bc70f8e464d60f8349af217e195a0a25